src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -94,6 +95,21 @@ class SeedVaultExportComponent extends Component {
         seed: '',
     };
 
+    /**
+     * Gets seed vault file path
+     *
+     * @method getPath
+     *
+     * @param {string} prefix
+     *
+     * @returns {string}
+     */
+    static getPath(prefix) {
+        return `${
+            isAndroid ? RNFetchBlob.fs.dirs.DownloadDir : RNFetchBlob.fs.dirs.CacheDir
+        }/${prefix}${moment().format('YYYYMMDD-HHmm')}.kdbx`;
```

Comment:
Should this have a dash after the prefix?
```suggestion
        }/${prefix}-${moment().format('YYYYMMDD-HHmm')}.kdbx`;
```

Reply:
To match the path in desktop implementation, yes.